tasks/deploy/publish.js

```diff
@@ -25,7 +25,10 @@ module.exports = function (gruntOrShipit) {
     function updateSymbolicLink() {
       shipit.log('Publishing release "%s"', shipit.releasePath);
 
-      var relativeReleasePath = path.join('releases', shipit.releaseDirname);
+      if ( shipit.config.releasesPath == undefined )
```

Comment:
if the user customize a release.path, i think we must configure the symbolic link accordingly... but you think that there's a better way?